.buildkite/pipeline.yml

```diff
@@ -32,20 +59,18 @@ steps:
         account_ids: "436617320021"
         no-include-email: true
         region: "us-west-2"
-      docker#v2.1.0:
+      docker#v3.1.0:
```

Comment:
I'm assuming you've tested this change and it works?  Sometimes upgrading versions will break existing code.

Reply:
Yep, the pipeline step is passing with the new docker plugin version so it's doing what it should.